diff --git a/node_modules/jest-util/build/specialChars.js b/node_modules/jest-util/build/specialChars.js
index 1f9ddcc..691e974 100644
--- a/node_modules/jest-util/build/specialChars.js
+++ b/node_modules/jest-util/build/specialChars.js
@@ -21,5 +21,5 @@ const ICONS = {
   todo: '\u270E'
 };
 exports.ICONS = ICONS;
-const CLEAR = isWindows ? '\x1B[2J\x1B[0f' : '\x1B[2J\x1B[3J\x1B[H';
+const CLEAR = isWindows ? '\x1B[2J\x1B[0f' : '';
 exports.CLEAR = CLEAR;
